src/libs/deploy.js

```diff
@@ -168,15 +169,57 @@ export const deployGoogle = async (signedApp, workspace = '/tmp/') => {
 };
 
 /**
- * Apple Store Deployment
+ * App Store Deployment
  *
- * @param {String} signedApp The name of the signed app
+ * @param {String} signedApp The name of the signed app, ONLY accepts .ipa / .pkg format
  * @param {string} [workspace='/tmp/'] The workspace to use
  * @returns The status of the deployment
  */
-// eslint-disable-next-line no-unused-vars
-export const deployAppleStore = async (signedApp, workspace = '/tmp/') => {
-  // TODO
+// eslint-disable-next-line import/prefer-default-export
+export const deployAppStore = async (signedApp, workspace = '/tmp/') => {
+  // Get app:
+  const signedAppPath = await fetchFileFromStorage(signedApp, workspace);
+  const signedAPP = require('fs').readFileSync(signedAppPath);
```

Comment:
I thought we tackled this issue already about using `import` and `promisify` rather than `require` and any sync functions?